ludwig/schema/combiners/utils.py

```diff
@@ -19,7 +19,13 @@ def get_combiner_jsonschema():
     return {
         "type": "object",
         "properties": {
-            "type": {"type": "string", "enum": combiner_types, "default": "concat"},
+            "type": {
+                "type": "string",
+                "enum": combiner_types,
+                "default": "concat",
+                "title": "type",
```

Comment:
If we're expecting this to be customer facing, should we capitalize `Type`?

Reply:
@martindavis all titles are lowercase in the schema - they get converted accordingly later